doc-bridgepoint/process/templates/ant_template.adoc

```diff
@@ -0,0 +1,88 @@
+= Title goes here
+
+xtUML Project Analysis Note
+
+NOTE:  Each section has a description that states the purpose of that section.
+Delete these section descriptions before checking in your note.  Delete this
+note as well.
+
+== 1 Abstract
+
+In this section, give a summary of the design that this note aims to
+describe.
+
+== 2 Introduction and Background
+
+In this section, outline the important points relating to this issue/bug that
+the reader would need to know in order to understand the rest of this
+document. Here is an example reference to a document reference <<dr-3>>.
+Here is an example with named text <<dr-3,SMASL Specification>>.
+Here we refer to the image below, <<Model-Explorer>>.
+
+[[Model-Explorer]]
+image::localimage.png[image description alternate text,width=25%]
+caption text under image
```

Comment:
I'll note also that in the MASL guide when you use the caption like I showed here and add `:xrefstyle: short` to the top of the document, I did my images like this:
```
.Project Preferences
image::images/image01.png[xtUML Project Preferences,id="proj_pref",width=50%] 
```
and then when I referenced in the text using &lt;&lt;proj_pref&gt;&gt; it would replace with the proper "Figure x" dynamically.  If I added a new image the cross ref rendering would figure it out automatically.

Reply:
changed caption but not cross reference
for now leaving cross reference to name the caption text